src/citrine/builders/design_spaces.py

```diff
@@ -55,19 +74,39 @@ def enumerate_simple_mixture_cartesian_product(
     balance_component: str
         name of the component that should make up the balance of the
         mixture (1-others). Must be a key in formulation_grids.
-    '''
+    descriptors: List[Descriptor]
+        design descriptors representing the degrees of freedom of the design
+        space. descriptors' keys should match the keys of formulation_grids
+    name: str
+        name for the EnumeratedDesignSpace
+    description: str
+        description for the EnumeratedDesignSpace
+    """
 
+    # Check that the passed balance component is in the grid keys
     assert balance_component in list(formulation_grids.keys()), \
         "balance component must be in formulation_grids' keys"
     non_balance_comps = [comp for comp in list(formulation_grids.keys())
                          if comp != balance_component]
+    non_balance_descriptors = [dd for dd in descriptors 
+                               if dd.key != balance_component]
+    non_balance_grids = {
+        k: v for k, v in formulation_grids.items()
+        if k in non_balance_comps
+        }
+    
```

Comment:
A bit of additional overhead here in order to use the `enumerate_cartesian_product()` function within this simple mixture function (need to create dummy descriptor list)